content/intro-curriculum/Before-The-Workshop.md

```diff
@@ -3,6 +3,7 @@ author: Jeffrey W. Hollister & Luke Winslow
 date: 9998-01-01
 slug: Before
 title: Before The Workshop
+draft: True
```

Comment:
Go ahead and take this off. Are you planning to update the screenshots too? Maybe when you update to 3.4, you could catch those all?